src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -117,6 +121,38 @@ impl VirtSandbox {
 
         Ok(resource_configs)
     }
+
+    async fn get_guest_details(&self) -> Result<()> {
```

Comment:
tbh this name is very ambiguous. You need some comments explaining this, or change this function name properly.

Reply:
This function wraps the agent ttrpc call `get_guest_detail`, and we can get the information of the guest OS. Maybe add some comments for this?

Reply:
Yeah I think comments are needed here. Probably further rename this together with the agent rpc handler is needed.